src/Resources/web/app_dev.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2017 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+use Contao\ManagerBundle\ContaoManager\Plugin as ManagerBundlePlugin;
+use Contao\ManagerBundle\HttpKernel\ContaoKernel;
+use Doctrine\Common\Annotations\AnnotationRegistry;
+use Symfony\Component\Debug\Debug;
+use Symfony\Component\Dotenv\Dotenv;
+use Symfony\Component\HttpFoundation\Request;
+
+// The check is to ensure we don't use .env in production
```

Comment:
But bundles are shifting over and generate the container config from env vars which is the way symfony wants to go.
So let's be an early adopter (which does not hurt at all).

I also want to stress out that the need for "dotenv" is higher on deployment than on dev machines because you usually have more control over the dev machine than over the deployment target (you can not set an env var on shared hosting apache that easy).

It will still produce side effects as simply grep'ing for "getenv" in my local vendor directory of a plain symfony installation gives many calls.

I simply fail to see why removing a cool feature from the non dev edition will do any harm when the dev edition shall benefit from it but am willing to get convinced if anyone provides good reasons.